src/main/java/com/fcgl/madrid/points/service/UserPointsService.java

```diff
@@ -1,22 +1,59 @@
 package com.fcgl.madrid.points.service;
 
+import com.fcgl.madrid.points.model.response.GetUserPointsResponse;
 import com.fcgl.madrid.points.model.UserPoint;
 import com.fcgl.madrid.points.repository.UserPointsRepository;
+import com.fcgl.madrid.points.model.InternalStatus;
+import com.fcgl.madrid.points.model.StatusCode;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.TransactionSystemException;
 
+import io.github.resilience4j.circuitbreaker.CallNotPermittedException;
+import io.github.resilience4j.circuitbreaker.annotation.CircuitBreaker;
+
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+import java.lang.Exception;
+import java.lang.Throwable;
+import java.lang.StringBuilder;
 
 @Service
 public class UserPointsService {
-    private UserPointsRepository userPointsRepository;
+	private UserPointsRepository userPointsRepository;
+
+	@Autowired
+	public UserPointsService(UserPointsRepository userPointsRepository) {
+		this.userPointsRepository = userPointsRepository;
+	}
+
+	@CircuitBreaker(name = "backendA", fallbackMethod = "fallback")
+	public List<UserPoint> findAll() {
+		return userPointsRepository.findAll();
+	}
+
+	public UserPoint getUserPointsById(Long userId) {
```

Comment:
The `@CircuitBreaker(name = "backendA", fallbackMethod = "fallback")` Should go above this function